src/test/regress/expected/follower_single_node.out

```diff
@@ -34,6 +34,17 @@ CREATE TABLE local(c int, d int);
 INSERT INTO test VALUES (1, 2), (3, 4), (5, 6), (2, 7), (4, 5);
 INSERT INTO ref VALUES (1, 2), (5, 6), (7, 8);
 INSERT INTO local VALUES (1, 2), (3, 4), (7, 8);
+-- Check repartion joins are supported
+SET citus.enable_repartition_joins TO ON;
+SELECT * FROM test t1, test t2 WHERE t1.x = t2.y ORDER BY t1.x;
```

Comment:
At this point, we've not added coordinator to the metadata or running it on a secondary. I guess this test is not useful here

Reply:
Above we have:

```sql
SELECT 1 FROM master_add_node('localhost', :master_port, groupid => 0);
SELECT 1 FROM master_set_node_property('localhost', :master_port, 'shouldhaveshards', true);
```

so isnt coordinator added?

Reply:
(I probably meant that we're not running it on the secondary at  this point -- given that the test file is intended for secondaries --, but lets keep this test in case we don't have something similar in other places)

Reply:
We also test this in `single_node.sql` so we can remove it maybe.